test/test_da.py

```diff
@@ -292,6 +307,8 @@ def test_unbalanced_sinkhorn_transport_class():
     Xs, ys = make_data_classif('3gauss', ns)
     Xt, yt = make_data_classif('3gauss2', nt)
 
+    Xs, ys, Xt, yt = from_numpy(nx, Xs, ys, Xt, yt)
```

Comment:
why not make the function nx.from_numpy and nx.to_numpy able to cpnvert multiple arrays (similarly to what you implemented in from_numpy ? This seems like a better idea than multiplying the function (and giving it a class)

Reply:
Done, now included in the backend class